tests/trainer/test_checkpoint.py

```diff
@@ -1189,6 +1221,95 @@ def _assert_expected_num_checkpoints(
         files = os.listdir(save_folder)
         assert len(files) == expected_num_files
 
+    @pytest.mark.parametrize('world_size', [
+        pytest.param(2, marks=pytest.mark.world_size(2)),
+    ])
+    @pytest.mark.parametrize('device', [
+        pytest.param('gpu', marks=pytest.mark.gpu),
+    ])
+    @pytest.mark.parametrize(
+        'use_orig_params,sync_module_states,model_1_init_device,model_2_init_device',
+        [
+            pytest.param(False, True, 'cpu', 'cpu'),
+            pytest.param(False, True, 'cpu', 'meta'),
+        ])
```

Comment:
can you remove the args which always have the same value?

Also I think it's fine to just leave both devices on meta / leave the default args in get_trainer